spec/data-spec.js

```diff
@@ -185,6 +189,13 @@ describe("Data", () => {
 				expect(+circle.attr("cx")).to.be.closeTo(expectedCx[i], 0);
 				expect(+circle.attr("cy")).to.be.closeTo(expectedCy[556][i], 1);
 			});
+
+			d3.selectAll('.c3-circles-778-889 .c3-circle').each(function (d, i) {
```

Comment:
ah, oki will fix. for some reason the tests do not work for me locally ( nor phantom nor chrome ).